labcontrol/db/tests/test_process.py

```diff
@@ -32,6 +33,7 @@
     LibraryPrep16SProcess, QuantificationProcess, PoolingProcess,
     SequencingProcess, GDNAPlateCompressionProcess, NormalizationProcess,
     LibraryPrepShotgunProcess)
+from labcontrol.db.sheet import Sheet, SampleSheet, SampleSheet16S, SampleSheetShotgun
```

Comment:
Line width longer than 80 characters.

I noticed that the latest flake8 no longer require line width <= 80 characters. Strange but interesting. I actually dislike this 80-char limit

Reply:
That's very interesting - thanks for the observation! I noticed that as well, and assumed it was a misconfiguration in Travis.

I will look into that - it seems like that would be a deviation from PEP8.